src/components/BuySell/BuySellWrapper.tsx

```diff
@@ -1,26 +1,43 @@
-import React, { useEffect } from 'react'
+import React, { useEffect, useState } from 'react'
 
 import TransactionWatcher from 'components/TransactionWatcher'
 import { BuySellWidget } from 'components/BuySell'
 import useTransactionWatcher from 'hooks/useTransactionWatcher'
 import useBuySell from 'hooks/useBuySell'
 
+import { ExchangeIssuanceWidget } from 'components/ExchangeIssuance'
+import useExchangeIssuance from 'hooks/useExchangeIssuance'
+
 interface BuySellWrapperProps {
   tokenId: 'index' | 'dpi' | 'cgi'
 }
 
 const BuySellWrapper: React.FC<BuySellWrapperProps> = ({ tokenId }: any) => {
   const { transactionStatus } = useTransactionWatcher()
   const { onSetBuySellToken } = useBuySell()
+  const { onSetIssuanceToken } = useExchangeIssuance()
+
+  const [isIssuance, setIsIssuance] = useState(false)
 
   useEffect(() => {
     onSetBuySellToken(tokenId)
   }, [onSetBuySellToken, tokenId])
 
-  return (
+  useEffect(() => {
+    onSetIssuanceToken(tokenId)
+  }, [onSetIssuanceToken, tokenId])
+
+  return !isIssuance ? (
```

Comment:
No need to use ternary here. Just if statements + returns will be cleaner. Especially if we add more cases later on.

Reply:
fixed